main.go

```diff
@@ -50,7 +50,9 @@ func run() error {
 			ProcessTickTimeout time.Duration `conf:"default:5s"`
 		}
 		Store struct {
-			ResetEmptyTickKeys bool `conf:"default:false"`
+			ResetEmptyTickKeys bool   `conf:"default:false"`
```

Comment:
what's this exactly?

Reply:
Flag to reset the empty tick count for every epoch and trigger a re-calculation. This is here to fix the counts in the event that archiver did not calculate them correctly.